### PR TITLE
A really simple script to auto install qFlipper

### PR DESCRIPTION
**Pull request by DXVVAY**
this is a really simple script to auto install qFlipper for lazy people like I'm not expecting this to get merged just thought that we haven't had a new badusb script in a while

# What's new

- [New added badusb script ]

### CODE REVIEW

File: assets/resources/badusb/qFlipper-windows.txt [hunk 1]
```diff
@@ -0,0 +1,20 @@
+DELAY 3000
+GUI r
+DELAY 1000
+STRING powershell (new-object System.Net.WebClient).DownloadFile('https://update.flipperzero.one/builds/qFlipper/1.2.2/qFlipperSetup-64bit-1.2.2.exe','%TEMP%\Dexv.exe'); Start-Process "%TEMP%\Dexv.exe"
```
Comment (dzienisz) [inline assets/resources/badusb/qFlipper-windows.txt]: I would not fix the version :(

Comment (DXVVAY) [reply assets/resources/badusb/qFlipper-windows.txt:4]: Wdym

Comment (dzienisz) [reply assets/resources/badusb/qFlipper-windows.txt]: you wrote 1.2.2 do you want to update this version everytime?

Comment (DXVVAY) [reply assets/resources/badusb/qFlipper-windows.txt]: I did think if that that I would be easier to just got to the versions and repeat TAB 10 times tills it gets the current version but I thought this is so much cleaner and there isn't so many qFlipper updates anyway but I think I will be updating it to the first solution I'll be trying it soon

**Comment by skotopes** [comment]
This PR is scheduled for next release, we are currently working on new indexer for update server that will provide `latest` version for download.

**Comment by skotopes** [comment]
@drunkbatya will update this PR when we deploy it.

**Comment by aprosvetova** [comment]
The latest version is actually already available (always has been): `https://update.flipperzero.one/qFlipper/release/windows-amd64/installer`

I suggest you rename the file to `install-qFlipper-windows.txt` for extra clarity

**Comment by DXVVAY** [comment]
> The latest version is actually already available (always has been): `https://update.flipperzero.one/qFlipper/release/windows-amd64/installer`
> 
> I suggest you rename the file to `install-qFlipper-windows.txt` for extra clarity

just updated the script so it installs the latest version and and changed the name

**Comment by DXVVAY** [comment]
I have just realized from someone sending me a message about it on discord that the script may cause a false positive from anti virus programs just to let you know if you want to avoid false positives.

**Comment by skotopes** [comment]
I can not sync this branch with dev. Also it looks like it got some merge artifacts.
Please open new PR

**Comment by DXVVAY** [comment]
> I can not sync this branch with dev. Also it looks like it got some merge artifacts. Please open new PR

sure

**Comment by DXVVAY** [comment]
> I can not sync this branch with dev. Also it looks like it got some merge artifacts. Please open new PR

made a new one

**Pull request closed without merging**